Scenes/BattleScenes/Battle.gd

```diff
@@ -1,15 +1,57 @@
 extends Node
+class_name Battle
+const script_name := "battle"
+
```

Comment:
Is the changes to this file just compatiability and renaming?

Reply:
script_name is used for logging and class_name is used for instancing although in this case it's unneeded (since it's only used in it's specific scene). I'll remove this in the next commit!